roles/common/tasks/main.yaml

```diff
@@ -3,11 +3,11 @@
 
 - name: Install packages
   apt:
-    name: ['git', 'apache2', 'mariadb-server', 'php8.1-fpm', 'php8.1-mysql',
-           'php8.1-curl', 'php8.1-memcache', 'php8.1-memcached', 'php8.1-intl',
-           'php8.1-mbstring', 'php8.1-xml', 'php8.1-zip', 'php8.1-apcu',
-           'php8.1-gd', 'php8.1-imagick',  'python3-certbot-apache',
-           'python3-mysqldb', 'sphinxsearch', 'mp3gain', 'unzip', 'memcached',
+    name: ['git', 'apache2', 'mariadb-server', 'php7.0-fpm', 'php7.0-mysql',
```

Comment:
Same here - we run Jammy in production and staging, where PHP 8.2 exists. You can't just change the version like that.

Reply:
When I installed LibriVox on Mantic and browsed my local installation via https://librivox.org/search, I got loads of PHP API deprecation errors and no data whatsoever. I will run this again on Jetty and see if I can make it work.

I have changed this pull request to a draft to indicate its WIP status. I will install this on Jetty and restore the file to its original state.

Reply:
I could indeed install the app on Jetty and I have restored the `roles/common/tasks/main.yaml` file to its original state.